### ISSUE TITLE
`Error: Wrong number of arguments` on master

### ISSUE THREAD

**Issue by torfjelde** [nnicandro/emacs-jupyter#434]
I'm currently getting the following error when I try to, well, most things, e.g. `jupyter-run-repl` and `jupyter-connect-repl` both result in:

```
Wrong number of arguments: #<module function at 0x7f6469a80ee0 with data 0x56146310a6a0>, 4
```

I'm currently trying to do some digging myself, but I'm not particularly familiar with the codebase so uncertain how useful I'll be.

EDIT: Some additional information.

```
IPython          : 8.9.0
ipykernel        : 6.21.1
```

on Python 3.10.

And both `emacs-jupyter` and `emacs-zmq` are on `#master`.

**Comment by torfjelde** [comment]
Okay, so if I revert to `emacs-zmq#38dc6c4` and `emacs-jupyter#7d20c0a` things are working again.

**Comment by torfjelde** [comment]
Note that the above comment isn't necessarily the minimal revision necessary; I only chose these because I these versions are working on a separate device of mine.

**Comment by malb** [comment]
I think `(zmq-send-encoded stdin msg nil zmq-DONTWAIT)` should be `(zmq-send-encoded stdin msg zmq-DONTWAIT)`

**Comment by nnicandro** [comment]
The problem should be fixed by 16cbda79167b4e2f2c6b61b218658f0f660d97f9, let me know if that doesn't do it.

**Comment by torfjelde** [comment]
Seems to be working now! Thanks for the quick reply @nnicandro !
